Kite SDK/PSPrintSDK/OLKiteABTesting.m

```diff
@@ -171,10 +172,25 @@ - (void)setupHidePriceTest{
                                 }];
 }
 
+- (void)setupOfferPayPalTest{
+    NSDictionary *experimentDict = [[NSUserDefaults standardUserDefaults] objectForKey:kOLKiteABTestOfferPayPal];
+    if (!experimentDict) {
+        experimentDict = @{@"Yes" : @0, @"No" : @1};
```

Comment:
I'm sure this should be 0.5 / 0.5 . Don't worry I'll change it when I merge but call me out if I'm wrong